dynamo/preprocessing/Preprocessor.py

```diff
@@ -225,22 +225,19 @@ def standardize_adata(self, adata: AnnData, tkey: str, experiment_type: str) ->
 
         adata.uns["pp"] = {}
         adata.uns["pp"]["norm_method"] = None
+        self.basic_stats(adata)
         self.add_experiment_info(adata, tkey, experiment_type)
         main_info_insert_adata("tkey=%s" % tkey, "uns['pp']", indent_level=2)
-        main_info_insert_adata("experiment_type=%s" % experiment_type, "uns['pp']", indent_level=2)
+        main_info_insert_adata("experiment_type=%s" % adata.uns["pp"]["experiment_type"], "uns['pp']", indent_level=2)
+
         main_info("making adata observation index unique...")
-        self.unique_var_obs_adata(adata)
```

Comment:
why this is removed? it is a good idea to unique gene names and cell names. otherwise it will lead to some potential issues later

Reply:
I removed this line because it is duplicated operation with line 240.
 if self.convert_gene_name:
            main_info("applying convert_gene_name function...")
            self.convert_gene_name(adata)
            main_info("making adata observation index unique after gene name conversion...")
            self.unique_var_obs_adata(adata)   <---------------- THIS!
            
However, since you want to keep the conditional operation for the function unique_var_obs_adata, I will retrieve this duplicated line again.